docs/content/latest/explore/ysql-language-features/data-types.md

```diff
@@ -377,3 +377,54 @@ yugabyte=# SELECT (item).name FROM on_hand WHERE (item).price > 0.99;
 (1 row)
 ```
 
+## Range Types
+
+Range data types represent a range of values of an element type. Range types are usually referred to as the subtype of the range. The subtype needs to follow a strict order because it must be well-defined regardless of the position of element values which can be within, before, or after a value range.
+
+YSQL supports the following range types:
+
+- `tsrange`, which corresponds to range of timestamp without time zone.
+- `tstzrange`, which corresponds to range of timestamp with time zone.
```

Comment:
I think we support all of:

```
int4range — Range of integer

int8range — Range of bigint

numrange — Range of numeric

tsrange — Range of timestamp without time zone

tstzrange — Range of timestamp with time zone

daterange — Range of date
```
The are tested in this file: https://github.com/yugabyte/yugabyte-db/blob/master/src/postgres/src/test/regress/expected/yb_pg_rangetypes.out

Note that we also support user-defined range types as tested there at the top with:
```
create type textrange as range (subtype=text, collation="C");
```
This is covered in pg docs in the `8.17.8. Defining New Range Types` section: (https://www.postgresql.org/docs/11/rangetypes.html)

Note: As those are more advanced it's ok to omit them and focus either only on tsrange types or only on primitive types here. But I suggest to quickly note them so that users don't get the impression that's _all_ we support.

Reply:
Applied these changes